src/corecel/cont/Label.cc

```diff
@@ -18,7 +18,7 @@ namespace celeritas
  */
 Label Label::from_geant(std::string const& name)
 {
-    static const std::regex final_ptr_regex{"0x[0-9a-f]{4,16}$"};
+    static const std::regex final_ptr_regex{"(.*)(0x[0-9a-f]{4,16}.+)"};
```

Comment:
Since we're not using the match result from the first expression, why is it needed? Also, I'm confused by the removal of the `$` and the addition of `.+`. This means we're now matching pointers in the middle of an expression? And this change also means we'll no longer match the name `Foo0x1234` and will match `Foo0x1234z`.... which doesn't seem right. Can you also add tests to the `Label` class (currently in `LabelIdMultiMap`) to check which expressions match?

Reply:
Reflected volumes have an extension added after the pointer, so e.g. in cms2018 there are names like `EHAWR0x7f4a8f353e00_refl` (or `EHAWR0x7f4a8f353e000x1e0cea0_refl` if we add an extra pointer when generating the unique name). I was trying to strip just the last pointer + extension (oops `.+` should have been `.*`), but maybe we should just strip everything after the first `0x` like Geant4?

Reply:
(and yes, made it a draft because I still need tests :))

Reply:
Sorry, I didn't see the draft when I started reviewing! 😅

Reply:
I modified `Label::from_geant()` to strip all trailing pointers and extensions and added a few more cases to the `Label` test -- let me know if you can think of any other tests that would be helpful.